readme.md

```diff
@@ -103,6 +103,20 @@ Start the spinner. Returns the instance.
 
 Stop and clear the spinner. Returns the instance.
 
+### .succeed()
+
+Stop the spinner, change it to a green `✔` and persists the `text`. See the gif below. Returns the instance.
+
+### .fail()
+
+Stop the spinner, change it to a red `✖` and persists the `text`. See the gif below. Returns the instance.
+
+### .stopAndPersist(icon)
```

Comment:
I think we should make `symbol` optional and just print nothing there if not supplied.